jerry-core/ecma/base/ecma-alloc.c

```diff
@@ -94,6 +97,26 @@ DECLARE_ROUTINES_FOR (getter_setter_pointers)
 DECLARE_ROUTINES_FOR (external_pointer)
 
 /**
+ * Allocate memory for ecma-property pair
+ *
+ * @return pointer to allocated memory
```

Comment:
In Jerry a failed allocation stops the engine. Other allocator functions has this return value specification, I didin't want to change the custom.